### PR TITLE
Create dataset spreadsheet template update instructions

### PR DESCRIPTION
**Pull request by rija**
First draft of the instructions to update the dataset spreadsheet templates.

# Pull request for issue: #1259

This is a pull request for the following functionalities:

* [x] The body of the instructions for updating the dataset spreadsheet template as a markdown file in the same directory
* [x] Made minor correction to [Giganet page](https://sites.google.com/gigasciencejournal.com/giganet/gigadb/curation/spreadsheet-upload-scripts) that explain how to upload spreadsheets
* [x] Updated the Giganet documentation to [reference](https://sites.google.com/gigasciencejournal.com/giganet/gigadb/curation/spreadsheet-upload-scripts) the new template updating instructions

## How to test?

Follow the instructions in the documentation, at the end there should be a new PR created (we can delete it later)

## How have functionalities been implemented?

l have change to the template in gigascience's gigadb-website checkout , then used Github Desktop to create a pull request and documented the process

### CODE REVIEW

**Comment by only1chunts** [comment]
@rija I think you have include the incorrect Giganet page URL in the above notes you have included a link to the excel-submission-sheet-guidelines instead of the how to guide on [spreadsheet-upload-scripts](https://sites.google.com/gigasciencejournal.com/giganet/gigadb/curation/spreadsheet-upload-scripts)

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+5. Then return to Github Desktop, you should see that the "Changes" column on the left is not empty and reflects the changes your made
+6. Create a new branch by clicking on the drop-down menu right of "Current branch" in the toolbar
+7. Click on "New branch", choose a meaningful name (e.g: incorrect-doi-hint), and click "Create branch"
+8. Then make sure that out of the two options shown, you select "Bring my changes to (name of the branch just created)"
+9. Click "Switch branch", "Current branch" in the toolbar should now show the name of the just created branch
+10. It is time to commit the changes by adding a summary and description to the commit message form at the bottom left, and then click "commit to (name of the just created branch)"
+11. The main pane will show a list of what to do next. Choose the top one (already highlighted) "Publish your branch" by clicking the "Publish branch" button
+12. The top option in the main pane is highlighted and says "Preview the pull request from you current branch", and has a button "Preview Pull Request" that you click
+13. a preview pane will pop up, check that it says:
```
Comment (kencho51) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md:26]: **typo:** `pane` should be `plane`.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    *  "commit into base:develop from (name of the branch you have created)
+    *  the main pane contain the change you want to publish and be reviewed
```
Comment (kencho51) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md:28]: typo: `pane` should be `plane`.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+4. On your computer, make the change you want to the file in `gigadb/app/tools/excel-spreadsheet-uploader/template`
+5. Then return to Github Desktop, you should see that the "Changes" column on the left is not empty and reflects the changes your made
+6. Create a new branch by clicking on the drop-down menu right of "Current branch" in the toolbar
+7. Click on "New branch", choose a meaningful name (e.g: incorrect-doi-hint), and click "Create branch"
+8. Then make sure that out of the two options shown, you select "Bring my changes to (name of the branch just created)"
+9. Click "Switch branch", "Current branch" in the toolbar should now show the name of the just created branch
+10. It is time to commit the changes by adding a summary and description to the commit message form at the bottom left, and then click "commit to (name of the just created branch)"
+11. The main pane will show a list of what to do next. Choose the top one (already highlighted) "Publish your branch" by clicking the "Publish branch" button
+12. The top option in the main pane is highlighted and says "Preview the pull request from you current branch", and has a button "Preview Pull Request" that you click
```
Comment (kencho51) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md:25]: typo: `pane` should be `plane`.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md (hunk 2 above)
Comment (kencho51) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md]: **nitpick:** The first letter of the sentence should be capitalized.
**typo:** Missing a closing `"` in line 27.
**typo:** `pane` should be `plane`.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+17. Click on "Create Pull Request" (if instead you see "Draft Pull Request", click the drop-down menu to bring the other option)
+18. in the "Projects" section in the right-hand sidebar, add the pull request to the current sprint project
+
+
+
+## Reviewing
+
+During the reviewing, comments may be exchange on the pull requests directly between reviewers and author.
+
+If additional changes to the spreadsheet are necessary, a number of rounds of steps 6, 7 and 8 are to be performed again. When the pull request has been approved by two members of the tech team, it will be merged to the codebase
```
Comment (kencho51) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md]: **typo:** steps 6-8 is about creating a new branch, shouldn't it be steps 9-10 which are about committing changes?

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Merging and sharing with the users
+
+When two developers have approved the pull request, We will merge the pull request and will send a general email announcing the change with description and instructions for downloading the file.
```
Comment (kencho51) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md]: **typo:** `We` should be `we`.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md (hunk 3 above)
Comment (rija) [reply gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md:25]: Hi @kencho51,
no, I really meant "pane" :-)
Screenshot 2023-12-05 at 10 26 00 AM

Also, see: https://ux.stackexchange.com/questions/124588/whats-the-difference-between-a-panel-and-pane

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md (hunk 4 above)
Comment (rija) [reply gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md]: Good spot @kencho51!
I was previously documenting using command line, and when switching to Github Desktop which had more steps, I forgot to change the steps number  in the reviewing sections. 
It's now fixed to refer to steps 9 to 11.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md (hunk 3 above)
Comment (kencho51) [reply gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md:25]: Ok 😄

**Review by kencho51 — APPROVED** [review]
Hi @rija,

LGTM, happy to approve now.

File: gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+12. The top option in the main pane is highlighted and says "Preview the pull request from you current branch", and has a button "Preview Pull Request" that you click
+13. A preview pane will pop up, check that:
+    *  it says "commit into base:develop from (name of the branch with your changes)"
+    *  the main pane contain the change you want to publish and be reviewed
+    *  At the bottom left, it should have a green tick "Able to merge"
+14. If all good, you can click "Create Pull Request"
+15. Your web browser will open on a new Pull request form on the Gigascience's gigadb-website repository
+16. Fill in the form as instructed inline
+17. Click on "Create Pull Request" (if instead you see "Draft Pull Request", click the drop-down menu to bring the other option)
+18. In the "Projects" section in the right-hand sidebar, add the pull request to the current sprint project
```
Comment (pli888) [inline gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md]: **suggestion**: Might be better re-written like this:

18. Add the pull request to the current sprint project by pasting its link into a new note in the Tasks To Do column. Sprint projects are listed in [Projects (classic)](https://github.com/orgs/gigascience/projects?type=classic).

**Review by pli888 — APPROVED** [review]
**praise**: The documentation looks good to me and can be approved after looking at my inline comment.

**Pull request merged**